crates/iota-indexer/src/indexer.rs

```diff
@@ -33,29 +33,12 @@ use crate::{
 pub struct Indexer;
 
 impl Indexer {
-    pub async fn start_writer(
-        config: &IngestionConfig,
-        store: PgIndexerStore,
-        metrics: IndexerMetrics,
-    ) -> Result<(), IndexerError> {
-        let snapshot_config = SnapshotLagConfig::default();
-        Indexer::start_writer_with_config(
-            config,
-            store,
-            metrics,
-            snapshot_config,
-            PruningOptions::default(),
-            CancellationToken::new(),
-        )
-        .await
-    }
```

Comment:
Isn't it used at all?

Reply:
We only use the `start_writer_with_config`